### ISSUE TITLE
Deploy failed in CI

### ISSUE THREAD

**Issue by hlinnaka** [neondatabase/neon#2264]
https://github.com/neondatabase/neon/runs/7810118718

```
TASK [post version to console] *************************************************
fatal: [zenith-us-stage-ps-3]: FAILED! => {"changed": true, "cmd": "INSTANCE_ID=$(curl -s [http://169.254.169.254/latest/meta-data/instance-id)\ncurl](http://169.254.169.254/latest/meta-data/instance-id)/ncurl) -sfS -d '{\"version\":  }' -X PATCH [http://console-staging.local/api/v1/pageservers/$INSTANCE_ID\n](http://console-staging.local/api/v1/pageservers/$INSTANCE_ID/n)", "delta": "0:00:00.061015", "end": "2022-08-12 16:37:36.621649", "msg": "non-zero return code", "rc": 22, "start": "2022-08-12 16:37:36.560634", "stderr": "curl: (22) The requested URL returned error: 500 Internal Server Error", "stderr_lines": ["curl: (22) The requested URL returned error: 500 Internal Server Error"], "stdout": "", "stdout_lines": []}
fatal: [zenith-us-stage-ps-2]: FAILED! => {"changed": true, "cmd": "INSTANCE_ID=$(curl -s [http://169.254.169.254/latest/meta-data/instance-id)\ncurl](http://169.254.169.254/latest/meta-data/instance-id)/ncurl) -sfS -d '{\"version\":  }' -X PATCH [http://console-staging.local/api/v1/pageservers/$INSTANCE_ID\n](http://console-staging.local/api/v1/pageservers/$INSTANCE_ID/n)", "delta": "0:00:00.064330", "end": "2022-08-12 16:37:36.766888", "msg": "non-zero return code", "rc": 22, "start": "2022-08-12 16:37:36.702558", "stderr": "curl: (22) The requested URL returned error: 500 Internal Server Error", "stderr_lines": ["curl: (22) The requested URL returned error: 500 Internal Server Error"], "stdout": "", "stdout_lines": []}
```
I'm not sure if this is already discussed somewhere, but I couldn't find an issue to track it..

**Comment by hlinnaka** [comment]
Is this related to https://github.com/neondatabase/neon/pull/2263, @cicdteam ?

**Comment by cicdteam** [comment]
> Is this related to #2263, @cicdteam ?

yes, that PR will fix the issue, i waiting for deploy to ensure everything is well

**Comment by cicdteam** [comment]
fixed by #2263
